content/applications/hr/payroll.rst

```diff
@@ -285,29 +440,29 @@ To make new default working hours, type the name for the new working hours in th
 Working Hours` field on the new structure type form. Click :guilabel:`Create and edit`. A default
 working hours form will pop up. The default working hours form has two sections, a general
 information section and a tab listing out all the individual working hours by day and time. When the
```

Comment:
I *think* I understand this suggestion. One issue is when the edits are done, line numbers change. I think you mean to add the last 2 sentences from the next edit to this section since it also applies? If I did not read this suggestion correctly, please let me know!